sample.go

```diff
@@ -12,14 +12,17 @@ import (
 // traceID.
 type Sampler func(id uint64) bool
 
-func neverSample(_ uint64) bool { return false }
+// NeverSample samples no traces
```

Comment:
NeverSample will always return false. If used by a service it will not allow the service to start traces but will still allow the service to participate in traces started upstream.